bin/tictactoe

```diff
@@ -1,3 +1,20 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic Tac Toe!"
+game = Game.new
+until game.won?
+  game.play
+end
+
+play_again = ""
+until play_again.downcase = "n" || play_again.downcase == "no"
+  puts "Would you like to play again?"
+  play_again = gets.strip
+  play_again.downcase == "y" || play_again.downcase == "yes"
+  game = Game.new
+  until game.won?
+    game.play
+  end
+end
```

Comment:
Prefer to encapsulate CLI logic into a class.

For example the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
  		 
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
   # ex:
  puts "Welcome to Tic Tac Toe!"
    game = Game.new
    until game.won?
      game.play
    end

    play_again = ""
    until play_again.downcase = "n" || play_again.downcase == "no"
      puts "Would you like to play again?"
      play_again = gets.strip
      play_again.downcase == "y" ||lay_again.downcase == "yes"
      game = Game.new
      until game.won?
        game.play
      end
    end
  end
end
```